test/superv/node_runner.cljs

```diff
@@ -0,0 +1,17 @@
+(ns superv.node-runner
+  (:require [cljs.nodejs]
+            [cljs.test]
+            [superv.async-test]))
+
+(defn run-tests
+  ([] (run-tests nil))
+  ([opts]
+   (cljs.test/run-tests (merge (cljs.test/empty-env) opts)
+                        'superv.async-test)))
+
+(defn -main [& args]
+  (cljs.nodejs/enable-util-print!)
+  (defmethod cljs.test/report [:cljs.test/default :end-run-tests] [_] (js/process.exit 0))
+  (run-tests))
+
+(set! *main-cli-fn* -main)
```

Comment:
Do you think it is necessary to have this? In theory node tests should run with `https://github.com/babashka/neil`. I am not really familiar with cljs so I am just asking.